lib/webpack.config.js

```diff
@@ -217,6 +217,106 @@ const base = ({ alias = {}, env, tsTranspileOnly }) => ({
   },
 })
 
+const storybook = () => ({
+  mode: 'development',
+  plugins: [
+    new webpack.DefinePlugin({
+      __STORYBOOK_ROOT__: JSON.stringify(process.__STORYBOOK_ROOT__),
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(png|gif|jpg|jpeg)$/,
+        use: nodeResolve('file-loader'),
+      },
+      {
+        test: /Cesium\.js$/,
+        use: [
+          {
+            loader: nodeResolve('exports-loader'),
+            options: { Cesium: true },
+          },
+          nodeResolve('script-loader'),
+        ],
+      },
+      {
+        test: /\.jsx?$/,
+        exclude: function(modulePath) {
+          if (modulePath.indexOf('catalog-ui-search') > -1) {
+            if (
+              modulePath.lastIndexOf('node_modules') >
+              modulePath.lastIndexOf('catalog-ui-search')
+            ) {
+              return true
+            }
+            return false
+          }
+          return modulePath.indexOf('node_modules') > -1
+        },
+        use: babelLoader([nodeResolve('babel-plugin-react-docgen')]),
+      },
+      {
+        test: /\.(eot|svg|ttf|woff|woff2)$/,
+        use: {
+          loader: nodeResolve('file-loader'),
+          options: {
+            name: 'fonts/[name]-[hash].[ext]',
+          },
+        },
+      },
+      {
+        test: /\.tsx?$/,
+        use: [
+          {
+            loader: nodeResolve('ts-loader'),
+            options: { allowTsInNodeModules: true },
+          },
+          {
+            loader: nodeResolve('react-docgen-typescript-loader'),
+            options: {
+              skipPropsWithoutDoc: true,
+              // The `transpileOnly` option is set to true because the
+              // storybook environment is meant for rapid iteration and
+              // not validation. We should not be using storybook to
+              // enforce typing.
+              transpileOnly: true,
```

Comment:
This is the wrong block. This should be in the options for `ts-loader`.

Reply:
:man_facepalming: